src/debugging/DebugHelper.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { CancellationToken, debug, DebugConfiguration, ExtensionContext, workspace, WorkspaceFolder } from 'vscode';

```

Comment:
>import [](start = 0, length = 6)

Is there any standard practice around organizing imports?

Reply:
The linter enforces alphabetical order of import statements by imported filename, then alphabetical order of imported types.